Library/Formula/gitlab-ci-multi-runner.rb

```diff
@@ -16,21 +18,44 @@ class GitlabCiMultiRunner < Formula
   end
 
   depends_on "go" => :build
+  depends_on "godep" => :build
 
-  def install
-    ENV["GOPATH"] = buildpath
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+      :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
 
+  resource "prebuilt.tar.gz" do
+    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.0.4/docker/prebuilt.tar.gz",
+      :using => :nounzip
+    sha256 "43dedd023672990e27289e97bf74f493576956418148f6227917b8511e8aadfd"
+  end
```

Comment:
Why are both the resource _and_ the docker dep being used now? Surely we only need one or the other. I thought from the other comment there were problems with using an external Docker dependency?